src/main/java/com/amihaiemil/docker/RtContainer.java

```diff
@@ -230,4 +231,29 @@ public void unpause() throws IOException {
             unpause.releaseConnection();
         }
     }
+
+    @Override
+    public int waitOn(final String state) throws IOException {
+        UncheckedUriBuilder urib = new UncheckedUriBuilder(
+            this.baseUri.toString().concat("/wait")
+        );
+        if(null == state || state.isEmpty()){
+            urib.addParameter("condition", state);
+        }
+
+        final HttpPost waiter = new HttpPost(urib.build());
+        waiter.setHeader(new BasicHeader("Content-Type", "application/json"));
```

Comment:
@ry99 I don't think you need to add this header here. I don't remember adding it in other parts of the library.

Reply:
removed